src/generic/PermGroups.jl

```diff
@@ -365,9 +365,9 @@ function _permstring(p::Perm, width::Integer)
       end
       out = "("*join(res, ")(")*")"
```

Comment:
Here the full string is still created despite the `io` argument elsewhere. Avoiding this would require writing each element of `res` to `io` after checking that the overall length doesn't overflow; but this is probably not a worthy complication.

Reply:
Ok, that's what I did ;) One could even avoid allocating the cycle string, but I'm not that compulsive at this moment ;P